src/usr/local/www/vpn_ipsec_settings.php

```diff
@@ -296,6 +314,18 @@ function maxmss_checked(obj) {
 	'It is recommended to keep this option to no, unless the exact implications are known and compatibility is required for such devices (for example, some SonicWall boxes).'
 );
 
+$section->addInput(new Form_Input(
+	'maxexchange',
+	'Maximum IKEv1 Phase 2 Exchanges',
+	'number',
+	$pconfig['maxexchange'],
+	['placeholder' => '3']
+))->setHelp(
+	'IKEv1 phase 2 rekeying for one VPN gateway can be initiated in parallel. By default only 3 parallel rekeys are allowed. ' .
+	'Too low values can break VPN connections with many tunnel definitions. ' .
+	'If unsure set this value to the VPN connection with the maximum number of tunnels.'
```

Comment:
Change this to: "If unsure, set this value to match the largest number of Phase 2 entries on any Phase 1."